app/models/queued_email.rb

```diff
@@ -372,12 +372,10 @@ def get_integers(keys, return_dict = false)
       @integers[qi.key.to_s] = qi.value.to_i
     end
     if return_dict
-      result = @integers
+      @integers
```

Comment:
I get nervous around this file. I don't understand it well and Coverage is only around ~80%.
And the changed lines are not covered.